storage/innobase/trx/trx0sys.cc

```diff
@@ -317,22 +317,11 @@ trx_sys_print_mysql_binlog_offset(void)
 
 #ifdef WITH_WSREP
 
+#include <wsrep_xid.h>
```

Comment:
This introduces unnecessary dependency between InnoDB and MySQL/wsrep code. Setting ```xid->formatID = -1``` in code below is enough to indicate that the XID is null, so ```WSREP_*``` macros and ```wsrep_xid_init()```  don't have to be included.